proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,26 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+enum AuthzPermissionChange {
+  // Grant the address trade permissions
+  GRANT = 0;
+  // Revoke trade permissions from the address
+  REVOKE = 1;
+}
+
+// Grants or revokes trade permissions to a given address via authz
+message MsgToggleTradeController {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stride/x/stakeibc/MsgToggleTradeController";
+
+  // Message signer (admin only)
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // Chain ID of the trade account
+  string chain_id = 2;
+  // Permission change (either grant or revoke)
+  AuthzPermissionChange permission_change = 3;
+  // Address of controller
```

Comment:
Should we call this operator instead of controller, to keep it consistent with stTIA/stDYM?

Reply:
done